src/Compilers/CSharp/Portable/Symbols/FileIdentifier.cs

```diff
@@ -89,9 +89,6 @@ public ImmutableArray<byte> FilePathChecksumOpt
         }
     }
 
-    public static FileIdentifier Create(SyntaxTree tree)
```

Comment:
I considered keeping this API and forcing `SourceReferenceResolver` as a second parameter. Either way seems fine to me.